lib/restAPIs/cubes.js

```diff
@@ -0,0 +1,124 @@
+const assert = require('assert');
+
+const RestUtil = require('../util/RestUtil');
+const buildCommonQuery = require('../util/buildCommonQueryParams');
+
+/*
+  https://demo.microstrategy.com/MicroStrategyLibrary/api-docs/index.html#/Cubes
+*/
+module.exports = class RestTopic extends RestUtil {
+  _getBase() {
+    return 'cubes';
+  }
+
+  getCubeDefinition(cubeId) {
+    assert(cubeId, 'No cubeId provided');
+
+    return this._makeRequest(`v2/${this._getBase()}/${cubeId}`, false, 'GET', this.getProjectHeader())
+    .then(result => this.throwIfFailed(result, 200));
+  }
+  createCubeInstance(cubeId, body, offset, limit, fields) {
```

Comment:
Tiny formatting request - new line between functions. Instead of:

```
func1() { 

}
func2() { 

} 
```

Keep a new line between methods:
```
func1() { 

}

func2() { 

} 
```